### ISSUE TITLE
Dydx perpetual - Unexpected error occured when listening to order book streams

### ISSUE THREAD

**Issue by mancap314** [hummingbot/hummingbot#5276]
**Describe the bug**
When running a `perpetual_market_making` strategy on dydx, I get the error:
```
Unexpected error occured when listening to order book streams
```
This error is recurring, but apparently does not prevent the bot to work.

**Steps To Reproduce**
1. Configure the dydx connector (with API and stark key)
2. create a `perpetual_making_strategy`
3. start this strategy

**Release version**
Docker on Ubuntu, version 1.3.0

**Attachments**
```
2022-05-01 21:11:32,846 - 1 - hummingbot.connector.derivative.dydx_perpetual.dydx_perpetual_api_order_book_data_source - ERROR - Unexpected error occurred when listening to order book streams. Retrying in 5 seconds...
Traceback (most recent call last):
  File "/home/hummingbot/hummingbot/connector/derivative/dydx_perpetual/dydx_perpetual_api_order_book_data_source.py", line 180, in listen_for_subscriptions
    async for ws_response in ws.iter_messages():
  File "/home/hummingbot/hummingbot/core/web_assistant/ws_assistant.py", line 69, in iter_messages
    response = await self._connection.receive()
  File "/home/hummingbot/hummingbot/core/web_assistant/connections/ws_connection.py", line 62, in receive
    msg = await self._read_message()
  File "/home/hummingbot/hummingbot/core/web_assistant/connections/ws_connection.py", line 79, in _read_message
    msg = await self._connection.receive(self._message_timeout)
  File "/home/hummingbot/miniconda3/envs/hummingbot/lib/python3.8/site-packages/aiohttp/client_ws.py", line 216, in receive
    raise RuntimeError("Concurrent call to receive() is not allowed")
RuntimeError: Concurrent call to receive() is not allowed
```

*WARNING: Do NOT publish any exchange API keys or your wallet's private key. Whoever has access to them may steal your assets!*

**Optional: your discord username:**
manuelc#5151

**Comment by sirEven** [comment]
Can confirm the issue on macOS running 1.3.0 from source.

**Comment by phpmac** [comment]
Have you found a solution?

**Comment by sirEven** [comment]
> Have you found a solution?

Error still persists on my end (just tested on version 1.5.0)

**Comment by afkj0hn** [comment]
Hi, having the same exact issue as @mancap314 and @sirEven. 

Running latest release - version 1.6.0 on macOS from source.

**Comment by phpmac** [comment]
when order_leves is 1 more,order will auto cancel

**Comment by phpmac** [comment]
> Hi, having the same exact issue as @mancap314 and @sirEven.
> 
> Running latest release - version 1.6.0 on macOS from source.

dydx

**Comment by jubalo93** [comment]
Hi, 
I'm facing the same issue but on spot_perpetual_arbitrage strategy
![image](https://user-images.githubusercontent.com/16908747/185071538-fd5a214b-669c-435e-9422-e4880118489d.png)

**Comment by tomnewbi** [comment]
Hi, 
I have the same problem. 
Running latest release - version 1.8
someone has found a solution ?

**Comment by gusreyes01** [comment]
I'm having the same issue 
Hi, having the same exact issue as @mancap314 and @sirEven.

Running latest release  on macOS from source.

**Comment by rapcmia** [comment]
Hi @mancap314 @afkj0hn @sirEven @gusreyes01 good day,
DyDx Perpetual has been refactored by one of our community devs and deployed the fix that will be available on the upcoming Hummingbot 1.10 release. You may want to wait for the release or try it on our development branch. Thank you
